### ISSUE TITLE
Typescript type declaration issue

### ISSUE THREAD

**Issue by w121211** [svgdotjs/svg.js#995]
# Bug report
Typescript version: 3.4.5
svg.js code: current master source code
Typescript compile error:

> node_modules/@svgdotjs/svg.js/svg.js.d.ts:424:9 - error TS2416: Property 'parse' in type 'PathArray' is not assignable to the same property in base type 'Array<PathCommand>'.

In file svg.js.d.ts, comment out the `parse(array: ArrayAlias<PathCommand> | PathArrayAlias): PathCommand[];` solves the issue, but this result the PathArrayAlias not in use.
```
...
class PathArray extends Array<PathCommand> {
        constructor();
        constructor(d: ArrayAlias<PathCommand> | PathArrayAlias);

        move(x: number, y: number): this;
        size(width: number, height: number): this;
        equalCommands(other: PathArray): boolean
        morph(pa: PathArray): this
        at(pos: number): PathArray
        // parse(array: ArrayAlias<PathCommand> | PathArrayAlias): PathCommand[];
        bbox(): Box;
        to(a: any): Morphable
    }
```

**Comment by maxailloud** [comment]
I am having the same issue, even though I am currently using Typescript 3.2.4.
Removing the line should work and not cause any issue in the typing as the parse method is already on the Array type, and is generic so the type is the one used here `Array<PathCommand>`.

**Comment by Fuzzyma** [comment]
Should be fixed with 3.0.13
